### DISCUSSION TITLE
When will the API documentation be released?

### DISCUSSION THREAD

**Discussion by ghost** [tutao/tutanota#5071] [Q&A]
https://tutanota.com/nl/faq#integration just says "we have plans", but where are they? Is there a timeline? I'd like to write a third-party client and documentation would be nice.

**Comment by spuxx-dev** [comment]
I'm also really interested in seeing a documentation on the API.

**Comment by wcerfgba** [comment]
I also vote for this to be prioritised. I know Tutanota have said they won't provide an IMAP bridge #196 , and if they don't want to provide an official one that's fine, but our emails are _our_ data and we should have full access to them. Personally I would benefit a lot from a bridge and being able to use a different client, so I can integrate with Zotero, backup my emails, and compose plaintext at 80 columns. I'm so glad to see <https://github.com/crepererum/tatutanatata> is now available for backing up, but as they state, they haven't been able to reverse the API, so they have to resort to webdriver instead. Making API access easy should be a priority if Tutanota cares about user rights, and they must have some documentation internally to be able to drive their own development.

**Comment by K0SS4** [comment]
I don't want create another thread so I will write it here. It would be great if the API had an option for adding, searching, deleting aliases for emails so I can manage it externally. Currently I have like 50 different aliases and managing it through the current UI is not really a great experience(cannot sort/search current aliases).

**Comment by AdityaGarg8** [comment]
API should be documented properly and users should be allowed to use it. Currently it looks like someone saying that trust me, my code is open source but at the same time have made the code difficult to understand and avoid people from using it.
